tests/modules/icgc-argo-workflows/score/download/main.nf

```diff
@@ -4,11 +4,22 @@ nextflow.enable.dsl = 2
 
 include { SCORE_DOWNLOAD } from '../../../../../modules/icgc-argo-workflows/score/download/main.nf'
 
-workflow test_score_download {
+// Test with data in rdpc-qa
+workflow test_score_download_rdpcqa {
     
     study_id = params.test_data['rdpc_qa']['study_id']
     analysis_id = params.test_data['rdpc_qa']['analysis_id']
-    analysis = params.test_data['rdpc_qa']['analysis']
+    analysis = file(params.test_data['rdpc_qa']['analysis'])
 
     SCORE_DOWNLOAD ( analysis, study_id, analysis_id )
 }
+
+// Test with user provide data
+workflow test_score_download_user {
```

Comment:
This is okay since these are designed to test the functions with user providing params. If the user want to use it, it is better they import the subworkflow into their workflow for proper usage.